### PR TITLE
Don't show pricing message for Companies created after Jun 25

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Check companyCreation date for pricing changes message
Do not show message if Company was created after Jun 25, 2019

## Motivation and Context
Those Companies were already on-boarded with the new pricing

## How Has This Been Tested?
Tested in the local environment and added unit tests for the directive

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: test/unit/common/directives/dtv-pricing-changes-message.tests.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  });
+
+  describe('alertVisible:', function() {
+    it('should not show alert if company is missing',function() {
+      compileDirective();
+
+      expect($scope.alertVisible()).to.be.false;
+    });
+
+    it('should not show alert if company is missing',function() {
```
Comment (ezequielc) [inline test/unit/common/directives/dtv-pricing-changes-message.tests.js]: @alex-deaconu Looks like this test is a duplicate from the above. Thanks!

Comment (alex-deaconu) [reply test/unit/common/directives/dtv-pricing-changes-message.tests.js]: Good catch, removed!

Comment (ezequielc) [reply test/unit/common/directives/dtv-pricing-changes-message.tests.js]: Thanks! 👍

**Pull request merged**